test_validate_dim_operation_invalid_parameters[image_dims2-4-add-TypeError-Array has incorrect dtype of 'int64'. The dtype must be a subtype of <class 'bool'>.] - AssertionError: Regex pattern did not match.

### Describe the bug, what's wrong, and what you expected.

On armhf the test is failing with:
419s tests/core/test_imagedata_filters.py::test_validate_dim_operation_invalid_parameters[image_dims2-4-add-TypeError-Array has incorrect dtype of 'int64'. The dtype must be a subtype of <class 'bool'>.] FAILED [  0%]
1805s FAILED tests/core/test_imagedata_filters.py::test_validate_dim_operation_invalid_parameters[image_dims2-4-add-TypeError-Array has incorrect dtype of 'int64'. The dtype must be a subtype of <class 'bool'>.] - AssertionError: Regex pattern did not match.
1805s  Regex: "Array\\ has\\ incorrect\\ dtype\\ of\\ 'int64'\\.\\ The\\ dtype\\ must\\ be\\ a\\ subtype\\ of\\ <class\\ 'bool'>\\."
1805s  Input: "Array has incorrect dtype of 'int32'. The dtype must be a subtype of <class 'bool'>."
1805s = 1 failed, 8691 passed, 146 skipped, 1144 deselected, 9 xfailed in 1424.61s (0:23:44) =

Full log: https://autopkgtest.ubuntu.com/results/autopkgtest-resolute/resolute/armhf/p/python-pyvista/20251203_214153_3bd48@/log.gz

### Steps to reproduce the bug.

Just call the testsuite

### System Information

```shell
Ubuntu devel :)
```

### Screenshots

_No response_

--- a/tests/core/test_imagedata_filters.py
+++ b/tests/core/test_imagedata_filters.py
@@ -1231,7 +1231,7 @@ def test_validate_dim_operation(
             4,
             operator.add,
             TypeError,
-            "Array has incorrect dtype of 'int64'. The dtype must be a subtype of <class 'bool'>.",
+            "The dtype must be a subtype of <class 'bool'>.",
         ),
         (
             (2, 2, 2),

